README.md

```diff
@@ -65,32 +65,31 @@ npm run fix-lint
 
 ## ASP.NET Demos
 
-1. You will need Windows. Enable Developer Mode so that the `mklink` command doesn't require elevation. Refer to https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/.
+1. You will need Windows. Enable Developer Mode so that the `mklink` command doesn't require elevation.\
+About symlinks: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/.\
+About Developer Mode: https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development.\
+To enable Developer Mode, type "Developer Settings" in a Windows search box and toggle the "Developer Mode" switch in the opened settings dialog.
 
-1. In the `mobile` repository:
-   - Run `PrepareWorkspace.cmd`.
-   - Then, run the following commands:
-     ```
-     cd GitHub
-     npm run build
-     ```
-     Or run `Packer.cmd`.
+1. In the `mobile` repository, use the Windows console to:
+   - 2.1 Run `PrepareWorkspace.cmd`.
+   - 2.2 Run `Packer.cmd`.
 
-1. Open the `repository.config.json` file and set the `hg` property to a path where your clone of the `mobile` repository is located, for example:
+1. Open the `mobile/repository.config.json` file and set the `hg` property to the path where your clone of the `mobile` repository is located. For example:
```

Comment:
The unnecessary "mobile/" prefix. The repository.config.json file is inside this repo.

Reply:
yeap, a especially added this prefix to show that this file is located on the repository's root